src/code_snippets/remove_repeated_characters.go.tmpl

```diff
@@ -0,0 +1,28 @@
+{{/*
+    Removes the repeated letters in a text like 'byyeeee' => 'bye'
+    Useful to add it in another more complex commands.
+
+    Author: Faculord <https://github.com/LattandiFacundo>
+*/}}
+
+{{ $toFix := "oooooooooooopppppppppppssssssss"}}
+{{ $exception := "o"}} {{/*Would keep two repeated characters if they matches the exception*/}}
```

Comment:
I don't mean to be pedantic, but - for a snippet - I don't feel that this is adequate. Snippets should be fairly easy to customize. For example, what if I wanted to have two exceptions? What if I wanted one character to have a limit of 3 and the other 2? (You may disagree, of course; let me know if you don't think what I'm saying makes sense.)